Installation/CMakeLists.txt

```diff
@@ -1298,3 +1298,24 @@ if(NOT CGAL_BRANCH_BUILD AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/doc")
   # in a non-branch build this is the top-level CMakeLists.txt
   add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/doc")
 endif()
+
+#print some info about versions
+if(RUNNING_CGAL_AUTO_TEST)
```

Comment:
```suggestion
if(RUNNING_CGAL_AUTO_TEST OR CGAL_TEST_SUITE)
```

Reply:
Because `Scripts/developer_scripts/run_testsuite_with_ctest` defines `CGAL_TEST_SUITE`. The name `RUNNING_CGAL_AUTO_TEST` is the old one defined by the legacy test scripts.